app/assets/javascripts/miq_application.js

```diff
@@ -731,8 +731,16 @@ function miqAjax(url, serialize_fields) {
 }
 
 // Function to generate an Ajax request for EVM async processing
-function miqAsyncAjax(url) {
-  miqJqueryRequest(url, {beforeSend: true});
+function miqAsyncAjax(url, serialize_fields) {
```

Comment:
In https://github.com/ManageIQ/manageiq/pull/11324, I'm trying to remove all the users of the `serialize_fields === true` branch of `miqAjaxButton` so that we always explicitly pass the object with the values we want sent.

So.. if you could *not* introduce the `true` variant here, I'd appreciate it.. :bow: ;)